website/docs/guides/creating-pages.md

```diff
@@ -86,6 +86,19 @@ You can use the full power of React in Markdown pages too, refer to the [MDX](ht
 
 :::
 
+## Available exports
```

Comment:
Thanks, but this is not an ideal place to document this, because this export is available for all mdx files, not just mdx pages

I suggest adding it here instead: https://docusaurus.io/docs/next/markdown-features/react

Reply:
Also worth adding an example to dogfood this and ensure it keeps working in practice, like

```mdx
Example:

<div>
  <div>__filename: {__filename}</div>
  <div>__dirname: {__dirname}</div>
  <div>toc: {JSON.stringify(toc)}</div>
  <div>frontMatter: {JSON.stringify(frontMatter)}</div>
</div>
```

This is just an example, making it look better is 👍